great_expectations/cli/suite.py

```diff
@@ -318,6 +325,44 @@ def suite_new(suite, directory, empty, jupyter, view, batch_kwargs):
         raise e
 
 
+@suite.command(name="delete")
+@click.option("--suite", "-es", default=None, help="Expectation suite name.")
+@click.option(
+    "--directory",
+    "-y",
+    default=None,
+    help="Delete expectation suite.",
+)
+def suite_delete(suite, directory):
+    """Delete specified expectation suite from data_context expectation store.
+
+    Args:
+        expectation_suite_name: The name of the expectation_suite to delete
+
+    Returns:
+        True for Success and False for Failure.
+    """
+    try:
+        context = DataContext(directory)
+    except ge_exceptions.ConfigNotFoundError as err:
+        cli_message("<red>{}</red>".format(err.message))
+        return
+
+    suite_names = context.list_expectation_suite_names()
+    if len(suite_names) == 0:
+        cli_message("No expectation suites found")
+        return
+
+    if len(suite_names) > 0:
+        expectation_suite = ExpectationSuite(expectation_suite_name=suite)
+        key = ExpectationSuiteIdentifier(expectation_suite_name=suite)
+        if key:
+            context.delete_expectation_suite(expectation_suite)
+        else:
+            cli_message("No matching expectation suites found")
+            return
```

Comment:
```suggestion
sys.exit(1)
```
To conform with the cli patterns

Reply:
Updated in latest commit